src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerdeletionservice/repository/OffenderDeletionRepository.kt

```diff
@@ -253,6 +282,16 @@ class OffenderDeletionRepository(
     }
   }
 
+  fun deleteAllOffenderDataExcludingBookings(offenderNumber: String): Set<Long>? {
+    log.info("Deleting all offender data for offender: '{}'", offenderNumber)
+    val offenderIds = offenderIdsFor(offenderNumber)
+    if (offenderIds.isEmpty()) { throw OffenderNotFoundException(offenderNumber) }
+    deleteNonBaseRecordOffenderData(offenderIds)
+    deleteBaseRecordOffenderData(offenderIds)
+    log.info("Deleted {} offender records with offenderNumber: '{}'", offenderIds.size, offenderNumber)
```

Comment:
have you thought about making these `customEvents` in app insights instead - seems like a natural candidate rather than just `log` messages

Reply:
actually ignore me - think you track it later on anyway